.github/workflows/renovate.yml

```diff
@@ -27,8 +25,14 @@ jobs:
           app-id: ${{ vars.BOT_APP_ID }}
           private-key: ${{ secrets.BOT_PRIVATE_KEY }}
       - env:
-          RENOVATE_TOKEN: ${{ secrets.RENOVATE_TOKEN || steps.generate-token.outputs.token }}
+          RENOVATE_TOKEN: ${{ steps.generate-token.outputs.token || secrets.RENOVATE_TOKEN }}
+          RENOVATE_GIT_AUTHOR: ${{ vars.RENOVATE_GIT_AUTHOR }}
```

Comment:
Seems there is no default value for `RENOVATE_GIT_AUTHOR` now, will it still work when using `PAT` instead of GitHub App Token?

Reply:
it will use the account of the pat as RENOVATE_GIT_AUTHOR.